src/include/iscsi-client/helps.rb

```diff
@@ -140,7 +141,15 @@ def initialize_iscsi_client_helps(include_target)
           ),
         # list of discovered targets
         "discovered"     => _(
-          "List of discovered targets. Start a new <b>Discovery</b> or <b>Connect</b> to any target."
+          "<p>This screen shows the list of discovered targets.<br>
+Use the <b>Discovery</b> button to get available iSCSI targets (from server specified in pop-up).<br>
```

Comment:
I wanted to say "from *a* server specified in *a* pop-up" but then I checked and found that it is a full size dialog, not a pop-up. So I'd rephrase to "Use the \<b>Discovery\</b> button to enter a server address and get available iSCSI targets for it.\<br>"